compose/container.py

```diff
@@ -131,10 +133,18 @@ def environment(self):
         return dict(var.split("=", 1) for var in self.get('Config.Env') or [])
 
     @property
+    def status(self):
+        return self.get('State.Status')
```

Comment:
Is this actually used anywhere? I don't think there's much point adding it if not.